.changeset/angry-scissors-give.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
Is this expected?

Reply:
Yes.  Perseus requires changeset files with every PR.  In this case it's a tooling only change so I used `yarn changeset add --empty` to add an empty changeset file.